sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlockBlobAsyncClient.java

```diff
@@ -58,14 +58,28 @@
 
     /**
      * Indicates the maximum number of bytes that can be sent in a call to upload.
+     * @deprecated Use {@link #MAX_STAGE_BLOCK_BYTES_LONG}
      */
+    @Deprecated
     public static final int MAX_UPLOAD_BLOB_BYTES = 256 * Constants.MB;
```

Comment:
Do we want to up these values to max int? I know they're deprecated but there's still a lot of room between 256MB and max int that customers can use if they want and their clamping logic would be misinformed from this.

Reply:
That's a good idea. However I'd rather do that only for blocks in this PR. Bumping single upload threshold deserve own PR and testing.

Reply:
Sadly. Looks like it would be considered as breaking change.
`[ERROR] Failed to execute goal org.revapi:revapi-maven-plugin:0.11.2:check (default) on project azure-storage-blob: The following API problems caused the build to fail:
[ERROR] java.field.constantValueChanged: field com.azure.storage.blob.specialized.BlockBlobAsyncClient.MAX_STAGE_BLOCK_BYTES: Constant field changed value from '104857600' to '2147483647'. (breaks semantic versioning)
[ERROR] java.field.constantValueChanged: field com.azure.storage.blob.specialized.BlockBlobClient.MAX_STAGE_BLOCK_BYTES: Constant field changed value from '104857600' to '2147483647'. (breaks semantic versioning)
[ERROR]
[ERROR] Consult the plugin output above for suggestions on how to ignore the found problems.`